### DISCUSSION TITLE
Automating Database Migrations in Pocketbase Containers

### DISCUSSION THREAD

**Discussion by uazapi** [pocketbase/pocketbase#5467] [Q&A] [answered]
Hello,

I am using Pocketbase as a framework to create my API in a containerized environment where each client has their own dedicated container. However, I've noticed that database migrations only execute after the admin logs in, which requires me to manually log in to each instance.

Is there any way to automate the migrations so that they are executed automatically when the container starts, without the need for manual login? Perhaps I have configured something incorrectly.

Thank you for your attention and for the excellent tool!

**Comment by ganigeorgiev** [comment, marked as answer]
> I've noticed that database migrations only execute after the admin logs in, which requires me to manually log in to each instance.

No, you are mistaken. New/unapplied migrations executes automatically on `serve` or on `migrate up`.

For more details please refer to:
- Go migrations - https://pocketbase.io/docs/go-migrations/ 
- JS migrations - https://pocketbase.io/docs/js-migrations/

Without more details how you build the containers and what is their deployment setup I don't think there is anything much I or anyone else can help with.
